content/tutorials/scripting-intro/code/BasicSimulation.java

```diff
@@ -15,29 +15,33 @@
  */
 
 //#full-example
-package computerdatabase;
+package example;
 
 import static io.gatling.javaapi.core.CoreDsl.*;
 import static io.gatling.javaapi.http.HttpDsl.*;
 
 import io.gatling.javaapi.core.*;
 import io.gatling.javaapi.http.*;
 
-public class ComputerDatabaseSimulation extends Simulation {
+public class BasicSimulation extends Simulation {
 
+  // Define HTTP configuration
+  // Reference: https://docs.gatling.io/reference/script/protocols/http/protocol/
   HttpProtocolBuilder httpProtocol =
-    http.baseUrl("https://computer-database.gatling.io")
-      .acceptHeader("application/json")
-      .contentTypeHeader("application/json");
+    http.baseUrl("https://api-ecomm.gatling.io")
+        .acceptHeader("application/json")
+        .userAgentHeader(
+            "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36");
```

Comment:
Removing extra whitespaces:
```suggestion
http.baseUrl("https://api-ecomm.gatling.io")
  .acceptHeader("application/json")
  .userAgentHeader("Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36");
```